cdc/processor/pipeline/sorter.go

```diff
@@ -220,7 +220,10 @@ func (n *sorterNode) start(
 					if err != nil {
 						return errors.Trace(err)
 					}
-
+					if msg.Row == nil {
+						log.Debug("message's row changed event is nil, it should be ignored", zap.Uint64("startTs", msg.StartTs))
```

Comment:
Perhaps the ignore operation here should be done in `mounter`, and feed the result back to the `sorter` through `decode` function.